src/compose/volumes/initializer/populateStorageAdapterRegistry.json

```diff
@@ -3,14 +3,14 @@
     "adapterid": "storage-adapter-mongodb",
     "url": "http://i40-aas-storage-adapter-mongodb:3100/submodels",
     "name": "mongo-adapter",
-    "submodelid": "sap.com/aas/submodels/part-100-device-information-model/sampleSM",
-    "submodelsemanticid": "opcfoundation.org/specifications-unified-architecture/part-100-device-information-model/sampleSM"
+    "submodelid": "sap.com/aas/submodels/part-100-device-information-model/10JF-1234-Jf14-PP22",
+    "submodelsemanticid": "opcfoundation.org/specifications-unified-architecture/part-100-device-information-model/10JF-1234-Jf14-PP22"
```

Comment:
Just remove /10JF-1234-Jf14-PP22. Only from semanticId not from id field.

Reply:
ok updated